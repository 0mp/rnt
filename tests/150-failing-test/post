. ../post.functions

trap cleanup EXIT

assert_file "test/exit.actual"
assert_file "test/err.actual"
assert_file "test/out.actual"
assert_file "test/exit.diff"
assert_file "test/err.diff"
assert_file "test/out.diff"
assert_same "expected/exit.diff" "test/exit.diff"
assert_same "expected/err.diff" "test/err.diff"
assert_same "expected/out.diff" "test/out.diff"

