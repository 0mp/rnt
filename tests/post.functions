cleanup()
{
  rm -f "$PWD/test/exit.actual"
  rm -f "$PWD/test/err.actual"
  rm -f "$PWD/test/out.actual"
  rm -f "$PWD/test/exit.diff"
  rm -f "$PWD/test/err.diff"
  rm -f "$PWD/test/out.diff"
}

assert_file()
{
  if test -e "${1:?}"; then
    echo "present: $1"
  else
    echo "missing: $1"
  fi
}

assert_nonexistent()
{
  assert_file ${1+"$@"}
}

assert_same()
{
  diff -u "${1:?}" "${2:?}"
}

