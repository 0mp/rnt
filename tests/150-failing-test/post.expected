present: test/exit.actual
present: test/err.actual
present: test/out.actual
present: test/exit.diff
present: test/err.diff
present: test/out.diff
